src/utilities/settings/hooks.tsx

```diff
@@ -0,0 +1,6 @@
+import {useContext} from 'react';
+import {SettingsContext} from './context';
+
+export function useSettings() {
+  return useContext(SettingsContext);
```

Comment:
Throw an exception if the context is falsy here - see the other hooks like sticky-manager for an example.

That'll ensure that this hook returns a value, and you won't need to do `settings != null` checks whenever you use the result of this hook